kibbeh/src/modules/layouts/GridPanels.tsx

```diff
@@ -29,7 +31,20 @@ export const MiddlePanel: React.FC<
   const screenType = useScreenType();
   return (
     <GridPanel>
-      <div className={!(screenType === "fullscreen" && !stickyChildren) ? `flex sticky top-0 w-full flex-col z-10 bg-primary-900 pt-5` : ""}>
+      <div
+        className={
+          !(screenType === "fullscreen" && !stickyChildren)
+            ? `flex sticky w-full flex-col z-8 bg-primary-900 pt-5`
+            : ""
+        }
+        style={
+          screenType === "fullscreen" &&
+          isElectron() &&
+          !useHostStore.getState().isLinux
```

Comment:
instead of putting this ugly check in multiple places, why don't you create a custom hook

Reply:
oh that's a great idea

Reply:
done